server/taskIngest.go

```diff
@@ -88,10 +88,46 @@ func ingestBillingDataForBillRepository(ctx context.Context, aaId, brId int) (er
 			"error":            err.Error(),
 		})
 	}
+	updateIdentityAndSubAccounts(ctx, aa)
```

Comment:
Note for the future: We should add another task to check sub accounts. This task would add missing sub accounts, remove obsolete ones and perhaps update their AwsIdentity field.
What do you think ?